AK/BitStream.h

```diff
@@ -121,6 +122,85 @@ class InputBitStream final : public InputStream {
     InputStream& m_stream;
 };
 
+class OutputBitStream final : public OutputStream {
+public:
+    explicit OutputBitStream(OutputStream& stream)
+        : m_stream(stream)
+    {
+    }
+
+    // WARNING: write aligns to the next byte boundary before writing, if unaligned writes are needed this should be rewritten
+    size_t write(ReadonlyBytes bytes) override
```

Comment:
Perhaps write_aligned would be a better name?

Reply:
unfortunately write is a pure virtual method im implementing from the abstract OutputStream, write_aligned would have been a better name if we were not constrained